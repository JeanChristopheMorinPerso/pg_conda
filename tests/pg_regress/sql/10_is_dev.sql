BEGIN;
CREATE EXTENSION pg_conda;

SELECT conda_is_dev('1.0.0') = FALSE;
SELECT conda_is_dev('1.0.0'::condaversion) = FALSE;
SELECT conda_is_dev('1.0.0.dev1') = TRUE;
SELECT conda_is_dev('1.0.0.dev1'::condaversion) = TRUE;
ROLLBACK;
