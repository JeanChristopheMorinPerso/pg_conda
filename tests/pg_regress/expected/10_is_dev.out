BEGIN;
CREATE EXTENSION pg_conda;
SELECT conda_is_dev('1.0.0') = FALSE;
 ?column? 
----------
 t
(1 row)

SELECT conda_is_dev('1.0.0'::condaversion) = FALSE;
 ?column? 
----------
 t
(1 row)

SELECT conda_is_dev('1.0.0.dev1') = TRUE;
 ?column? 
----------
 t
(1 row)

SELECT conda_is_dev('1.0.0.dev1'::condaversion) = TRUE;
 ?column? 
----------
 t
(1 row)

ROLLBACK;
